### DISCUSSION TITLE
browsing non rooted output

### DISCUSSION THREAD

**Discussion by stableartai** [zanllp/infinite-image-browsing#696] [Ideas]
Currently the image tiles and previews and searches only work with rooted output. When navigating to another output folder elsewhere, like on a different drive or any other path, there are now thumbnail previews unless you got to the exact folder with images.

**Comment by zanllp** [comment]
I didn't quite understand your point, but I guess you might be looking for something like this: If you want to search for files in other directories, you must add the target folder to the search scan path.


<img width="1257" alt="image" src="https://github.com/user-attachments/assets/2d2ec6ee-11b5-47f1-b386-90dcd867ef1d">

**Comment by zanllp** [comment]
<img width="533" alt="image" src="https://github.com/user-attachments/assets/2d13101c-11aa-418e-b579-43ed9a1f8c00">

**Reply by stableartai** [reply]
Ok thanks the add to search scan seem to have worked

**Reply by stableartai** [reply]
Maybe you call also help: why doesn't the sdxl style not show as searchable category. we have notice that scripting also does have options for sdxl which is part of sD 1.5 with sdxl 1.0

**Reply by zanllp** [reply]
Please provide your image source file.
